docs/spec_draft.md

```diff
@@ -2146,6 +2149,111 @@ where `predicate = rank(pred) == 0 ? pred : pred[i0, ..., iR-1]`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.shift_left
+
+### Semantics
+
+Performs element-wise left-shift operation on the `lhs` tensor by `rhs` number
+of bits and produces a `result` tensor.
+
+### Inputs
+
+| Name  | Type                   |
+|-------|------------------------|
+| `lhs` | tensor of integer type |
+| `rhs` | tensor of integer type |
+
+### Outputs
+
+| Name     | Type                   |
+|----------|------------------------|
+| `result` | tensor of integer type |
+
+### Constraints
+
+  * (C1) `lhs`, `rhs`, and `result` have the same type.
+
+### Examples
+
+```mlir
+// %lhs: [-1, -2, 3, 4, 7, 7]
+// %rhs: [1, 2, 3, 6, 7, 8]
+%result = "stablehlo.shift_left"(%lhs, %rhs): (tensor<6xi8>, tensor<6xi8>) -> tensor<6xi8>
+// %result: [-2, -8, 24, 0, -128, 0]
+```
+
+[Back to Ops](#index-of-ops)
+
+## stablehlo.shift_right_arithmetic
+
+### Semantics
+
+Performs element-wise arithmetic right-shift operation on the `lhs` tensor by
+`rhs` number of bits and produces a `result` tensor.
+
```

Comment:
It might be handy to have a bit-representational example to distinguish the difference between arithmetic and logical right shift. Wdyt @burmako?

Reply:
Let's figure this out as part of #289 and then apply uniformly to all examples in the spec. Meanwhile, I propose we leave these examples as is.